app/controllers/wrangling_guidelines_controller.rb

```diff
@@ -72,4 +72,13 @@ def destroy
 
     redirect_to(wrangling_guidelines_url)
   end
-end
\ No newline at end of file
+
+   private
+  def wrangling_guidelines_params
```

Comment:
Style note: it makes more sense to name the method `wrangling_guideline_params`, as in general we're talking about one generic wrangling guideline, or one specific wrangling guideline being updated.

Content note:
- we already have an admin only `before` filter, so having an `if` branch doesn't seem to add any value in this method
- this code is not functionally equivalent to what we had before; before we were protecting `content_sanitizer_version ` from being updated through the form, now we're permitting everything, including any future attributes to be added to the model.

The right way to implement strong params is to list out the params that we really want to permit to be set through the form, in this case `title` and `content`.